pylint_odoo/checkers/modules_odoo.py

```diff
@@ -149,6 +161,24 @@ def _check_duplicate_id_csv(self):
             return False
         return True
 
+    def _check_tabs_instead_of_spaces(self):
+        """Check tabs character instead of four spaces.
+        :return: False if exists errors and
+                 add list of errors in self.msg_args
+        """
+        self.msg_args = []
+        for type_file in self.config.extfiles_to_lint:
+            for ext_file in self.filter_files_ext(type_file, relpath=False):
+                countline = 0
+                with open(ext_file, 'rb') as fp:
+                    for line in fp:
+                        countline += 1
+                        line and line[0] == '\t' and self.msg_args.append(
```

Comment:
You should check for any tab inside the line. Not only at the beginning. You can have four spaces, then a tab, then other four spaces...